internal/xds/rbac/rbac_engine.go

```diff
@@ -131,9 +139,37 @@ func newEngine(config *v3rbacpb.RBAC) (*engine, error) {
 		}
 		policies[name] = matcher
 	}
+
+	auditOptions := config.GetAuditLoggingOptions()
+	auditLoggers := []*audit.Logger{}
+	if auditOptions != nil {
+		for _, logger := range auditOptions.LoggerConfigs {
+			auditLoggerFactory := audit.GetLoggerBuilder(logger.AuditLogger.Name)
+			if auditLoggerFactory == nil {
+				if logger.IsOptional {
+					continue
+				}
+				return nil, fmt.Errorf("No builder registered for %v", logger.AuditLogger.Name)
+			}
+			customConfig := logger.AuditLogger.TypedConfig
+			jsonConfig, err := json.Marshal(&customConfig)
+			if err != nil {
+				return nil, fmt.Errorf("could not convert audit logger custom config to json: %v", err)
+			}
+			auditLoggerConfig, err := auditLoggerFactory.ParseLoggerConfig(jsonConfig)
+			if err != nil {
+				return nil, fmt.Errorf("audit logger custom config did not match LoggerConfig: %v", err)
```

Comment:
Not sure if `did not match` is accurate. It could be a valid config format for this particular logger but some field has wrong value.

Reply:
Suggest change to `audit logger custom config could not be parsed`?

Reply:
Sounds good to me!